.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Issue #, if available:
+
+**Description of Changes**
+
+[//]: #  (A description of the change that you made and the new user experience that it creates)
+
+**Description of how you validated changes**
+
+[//]: #  (A description of you validated your changes and any relevant logs that show your change works)
+
+
+**Checklist**
+
+- [ ] If this change would make any existing documentation invalid, I have included those updates within this PR
+- [ ] I have added unit tests that prove my fix is effective or that my feature works
+- [ ] I have linted my code before raising the PR
```

Comment:
This point should be covered by our PR actions - improperly formatted code should fail the checks

Reply:
It will be but those are run after the PR has been raised and take several minutes to run so the PR is probably not going to sit around and wait for them to pass. This could save some turnaround time between the requestor and the reviewers.